integration-tests/tests/run-workflow.js

```diff
@@ -46,44 +46,8 @@ const testRunWorkflowFn = _.flow(
   await findInDataTableRow(page, testEntity.name, testEntity.attributes.input)
 })
 
-const launchWorkflowAndWaitForSuccess = async page => {
```

Comment:
Moved this out into a separate file (`utils/workflow-utils.js`). Previously, `run-workflow-on-snapshot` was requiring it from here. Now, requiring this module has a side effect since it calls `registerTest`.

This will conflict with #3063.